src/util/semvar.js

```diff
@@ -20,8 +21,9 @@ async function checkSemanticVersion() {
     remote = remote.trim();
     if (remote !== version) {
       console.warn(chalk`{whiteBright A newer version of Archway CLI is available ({green.bold v${remote}})\nSupport for {green.bold v${version}} has ended. Install the latest version with {yellow.bold npm install -g @archwayhq/cli}}`);
+      console.info('If you want skip this check, add ARCHWAY_SKIP_VERSION_CHECK=true to your environment file (.bashrc, .zshrc, ...)');
```

Comment:
technically you don't need to add... could just prepend it in the command or export the variable, but it is fine